root/tag/DeleteTag.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import FormSubmit from '../components/FormSubmit';
+import TagLink from '../static/scripts/common/components/TagLink';
+import bracketed from '../static/scripts/common/utility/bracketed';
+
+import TagLayout from './TagLayout';
+
+type Props = {
+  +$c: CatalystContextT,
+  +deleteDownvoted?: boolean,
+  +tag: TagT,
+};
+
+const DeleteTag = ({
+  $c,
+  deleteDownvoted = false,
+  tag,
+}: Props): React.Element<typeof TagLayout> | null => {
+  const title = deleteDownvoted
+    ? l('Delete tag downvotes')
+    : l('Delete tag uses');
+  const user = $c.user;
+
+  return user ? (
+    <TagLayout
+      $c={$c}
+      page="delete"
+      tag={tag}
+      title={title}
+    >
+      <h2>{title}</h2>
+      <p>
+        {exp.l(
+          deleteDownvoted ? (
+            `Are you sure you want to remove
+             all your votes against the tag “{tag}”?`
+          ) : (
+            `Are you sure you want to remove
+             all your uses of the tag “{tag}”?`
```

Comment:
I think this message confused me, because I was looking for a way to delete only upvotes, and thought this would delete all votes.  But I don't know if replacing "uses" with "upvotes" would be more confusing or less.

Reply:
"all your votes for the tag" has the same issue, since it's not clear it's for vs against. "all your votes in favor of the tag"? Too weird?

Reply:
We do use "upvotes" in the vote filters on user tag pages now, so maybe that's not so bad.  "uses (upvotes)" might work too.